mmv1/products/dlp/terraform.yaml

```diff
@@ -21,6 +21,33 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       deidentifyConfig.infoTypeTransformations.transformations.primitiveTransformation.replaceWithInfoTypeConfig: !ruby/object:Overrides::Terraform::PropertyOverride
         custom_flatten: templates/terraform/custom_flatten/object_to_bool.go.erb
         custom_expand: templates/terraform/custom_expand/bool_to_object.go.erb
+      deidentifyConfig.recordTransformations.fieldTransformations.condition.expressions.conditions.conditions.value.timestampValue: !ruby/object:Overrides::Terraform::PropertyOverride
+          validation: !ruby/object:Provider::Terraform::Validation
+            function: 'validateRFC3339Date'
+      deidentifyConfig.recordTransformations.fieldTransformations.condition.expressions.conditions.conditions.value.timeValue.hours: !ruby/object:Overrides::Terraform::PropertyOverride
+          validation: !ruby/object:Provider::Terraform::Validation
+            function: 'validation.IntBetween(0, 24)' # "An API may choose to allow the value "24:00:00" for scenarios like business closing time."
```

Comment:
would it make sense to have these validations for deidentifyConfig.recordTransformations.fieldTransformations.primitiveTransformation.replaceConfig.newValue as well?

Reply:
Definitely - add them in [b8f25c4](https://github.com/GoogleCloudPlatform/magic-modules/pull/6872/commits/b8f25c4586e31901aa7dbe6ae40ce92dcb1ee144)

Should the same validation be added for the existing infoTypeTransformations.transformations.primitiveTransformation.replaceConfig field ([link here](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/dlp/api.yaml#L1010))? I don't think it would be a breaking change, as any configurations not matching the validation rules would have API errors

Reply:
Probably that would be fine but I'd recommend doing it as a separate change because it'll need extra scrutiny - sometimes the API doesn't actually match the validation rules we think it does and we end up breaking users. I'd rather not couple this PR to it in case it ends up needing to be reverted or something.